apps/rush-lib/src/cli/test/__snapshots__/CommandLineHelp.test.ts.snap

```diff
@@ -375,7 +375,7 @@ exports[`CommandLineHelp prints the help for each action: publish 1`] = `
                     [--release-folder FOLDER] [--include-all]
                     [--version-policy POLICY] [--prerelease-name NAME]
                     [--partial-prerelease] [--suffix SUFFIX] [--force]
-                    [--apply-git-tags-on-pack]
+                    [--apply-git-tags-on-pack] [-c COMMITID]
```

Comment:
`COMMIT_ID` is a more readable name than `COMMITID`.  (Compare with `VERSION_POLICY_NAME` in this file.)

Reply:
Thanks! This is much more readable, I agree. For some reason, I thought the UTs didn't allow `_`s.